app/qml/map/MMMapController.qml

```diff
@@ -238,6 +251,13 @@ Item {
           root.nothingIdentified()
         }
       }
+      else if ( root.state === "select" )
+      {
+        let screenPoint = Qt.point( point.x, point.y )
+        let pair = identifyKit.identifyOne( screenPoint )
```

Comment:
How about putting the two calls at the beginning of the function so we do not need to duplicate them here?

Reply:
It will still need to be within a check for the two `root.state`s, as we wouldn't want to spend the time identifying for the other states